.github/workflows/imaginary.yml

```diff
@@ -11,27 +11,63 @@ jobs:
     name: Create Imaginary Container
 
     steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+        with:
+          repository: h2non/imaginary
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: env.BUILDKIT_STEP_LOG_MAX_SIZE=-1
+
       - name: Log in to GitHub Container Registry
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Setup Go
+        uses: actions/setup-go@v3
 
-      - name: Check out the repo
+      # https://github.com/WordPress/openverse-api/blob/f0c44d571249a2631eef5a13ea0d56702210daf3/.github/workflows/build_imaginary.yml#L47
+      - name: Update modules # https://github.com/h2non/imaginary/issues/387
         run: |
-          git clone https://github.com/h2non/imaginary.git
+         sed -i 's/bimg v1.1.4/bimg v1.1.9/' go.mod
+         go mod tidy
+      
+      # https://github.com/WordPress/openverse-api/blob/f0c44d571249a2631eef5a13ea0d56702210daf3/.github/workflows/build_imaginary.yml#L53
+      - name: Skip tests # https://github.com/golang/go/issues/29948
+        run: |
+         sed -i 's/RUN go test/# RUN go test/' Dockerfile
+         sed -i 's/RUN golangci/# RUN golangci/' Dockerfile

```

Comment:
is it possible to not skip the tests and golcangci?

Reply:
Then it will crash, see here: https://github.com/Z0ey2022/test/runs/7897992621?check_suite_focus=true

Reply:
I've started a test run with only `go test` disabled but golangci enabled: 
https://github.com/Z0ey2022/test/runs/7916361906?check_suite_focus=true

Reply:
Both needs to be disabled, or it will crash

Reply:
All right! Thanks for verifying!